src/index.html

```diff
@@ -6,17 +6,69 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>MOYO header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
       rel="stylesheet"
-      href="./style.css"
-    />
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images\logo.png"
+          alt="MOYO company logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#apple" class="nav-link is-active">Apple</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#samsung" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#smartphones" class="nav-link">Smartphones</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#laptops&computers"
```

Comment:
The `href` attribute of the `a` tag contains a space, which is not a good practice. Spaces should be replaced with `%20` in URLs. So it should be `href="#laptops%26computers"` instead of `href="#laptops&computers"`.